src/components/Alert/__stories__/index.stories.tsx

```diff
@@ -99,3 +99,20 @@ IconSize.decorators = [
     </div>
   ),
 ]
+
+export const AdvancedChildren = Template.bind({})
+AdvancedChildren.parameters = {
+  docs: {
+    storyDescription:
+      'If you want to add more information into your alert you can simply pass it to Alert component as children.',
+  },
+}
+AdvancedChildren.decorators = [
+  () => (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: 16 }}>
```

Comment:
Not needed ?

Reply:
@chambo-e Yes bad copy and paste, I changed it :)

Reply:
@chambo-e Yes bad copy and paste, I changed it :)